build.gradle

```diff
@@ -196,9 +196,9 @@ signing {
 
 dependencies {
     api platform('com.exactpro.th2:bom:4.5.0')
-    implementation 'com.exactpro.th2:grpc-check1:4.3.0-dev'
-    implementation 'com.exactpro.th2:common:5.6.0-dev'
-    implementation 'com.exactpro.th2:common-utils:2.2.0-dev'
+    implementation 'com.exactpro.th2:grpc-check1:4.4.0-multisubmit-dv4-7567351808-82885f6-SNAPSHOT'
+    implementation 'com.exactpro.th2:common:5.7.1-dev'
```

Comment:
5.7.2-dev is the latest version, I think we can use it here

Reply:
some unit tests fail with it

Reply:
Okay. Did you check which unit tests and why they failed?

Reply:
fixed (missed `scope` value in tests)